pandaharvester/harvestermessenger/shared_file_messenger.py

```diff
@@ -285,6 +285,8 @@ def feed_jobs(self, workspec, jobspec_list):
         pandaIDs = []
         for jobSpec in jobspec_list:
             accessPoint = self.get_access_point(workspec, jobSpec.PandaID)
+            if not os.path.exists(accessPoint):
```

Comment:
setup_access_points() does the same

https://github.com/PanDAWMS/panda-harvester/blob/master/pandaharvester/harvestermessenger/shared_file_messenger.py#L465

and it is called before feed_jobs() in submitter

https://github.com/PanDAWMS/panda-harvester/blob/master/pandaharvester/harvesterbody/submitter.py#L191

So I'm not sure why os.mkdirs() is required here.